linera-service/src/linera/main.rs

```diff
@@ -903,6 +903,7 @@ impl Runnable for Job {
                 let bytecode_id = context
                     .publish_bytecode(&chain_client, contract, service)
                     .await?;
+                println!("[ndr] BytecodeId: {:#?}", bytecode_id);
```

Comment:
Please log human strings to stderr. Stdout is for scripting

Reply:
This is a debug log that will be removed ha sorry
Trying to figure something out with the CI

Reply:
ok :)